docs/monitoring/05-02-grafana.md

```diff
@@ -17,6 +17,11 @@ This table lists the configurable parameters, their descriptions, and default va
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
 | **env.GF_USERS_DEFAULT_THEME** | Specifies the background color of the Grafana UI. You can change it to `dark`. | `light` |
-| **env.GF_USERS_AUTO_ASSIGN_ORG_ROLE** | Specifies the auto-assigned [user role](https://grafana.com/docs/grafana/latest/manage-users). You can change the value to `Viewer` or `Admin`. | `Editor' |
+| **env.GF_AUTH_GENERIC_OAUTH_ENABLED** | Enables the generic OAuth plugin of Grafana, being pre-configured with a configuration based on the in-cluster dex setup. | `true` |
+| **env.GF_USERS_AUTO_ASSIGN_ORG_ROLE** | Specifies the auto-assigned [user role](https://grafana.com/docs/grafana/latest/manage-users) for an user authenticated by Grafana. You can change the value to `Viewer` or `Admin`. | `Editor` |
+| **env.GF_AUTH_ANONYMOUS_ENABLED** | Enables anonymous login to Grafana. | `false` |
+| **env.GF_AUTH_ANONYMOUS_ORG_ROLE** | Specifies the auto-assigned [user role](https://grafana.com/docs/grafana/latest/manage-users) for an anonymous user. You can change the value to `Viewer` or `Admin`. | `Editor` |
 | **env.GF_LOG_LEVEL** | Specifies the log level used by Grafana. Be aware that logs at the `info` level may print logins, which can potentially be users' email addresses. | `warn` |
 | **persistence.enabled** | Specifies whether user and dashboard data used by Grafana is durably persisted. If enabled, the Grafana database will be mounted to a PersistentVolume and survive restarts. When using Grafana in a high-available setup using an external database, ensure that this flag is set to `false` | `true` |
+| **env.GF_ALERTING_ENABLED** | Enables the alerting feature of Grafana | `true` |
+| **env.GF_DASHBOARDS_MIN_REFRESH_INTERVAL** | Configures the minimum refresh interval for dashboards to prevent users from setting the dashboard refresh interval of a lower than given interval. Use the [official syntax](https://grafana.com/docs/grafana/latest/administration/configuration/#min_refresh_interval). | `10s` |
```

Comment:
```suggestion
| **env.GF_DASHBOARDS_MIN_REFRESH_INTERVAL** | Configures the minimum refresh interval for dashboards to prevent users from using a value lower than the given interval. Use the [official syntax](https://grafana.com/docs/grafana/latest/administration/configuration/#min_refresh_interval) to set this value. | `10s` |
```